### ISSUE TITLE
[ Bug ] Text widgets are placed at the top, in the Hub node 

### ISSUE THREAD

**Issue by JorgeR81** [Trung0246/ComfyUI-0246#47]
The text widgets are not placed correctly in the Hub node.  

---

![h1](https://github.com/user-attachments/assets/cfd33190-1596-426d-ad30-9fa012c4b863)

**Comment by JorgeR81** [comment]
If we zoom in a little, the text widget goes way.

 But it's still not displayed in the Hub, in the correct area.

![z2tb](https://github.com/user-attachments/assets/20c989c8-fb8b-4c02-bebd-d14f1a82828f)

![h1b](https://github.com/user-attachments/assets/a2363975-e8fa-45de-ac0c-157a0dd63a98)

**Comment by Trung0246** [comment]
Hm weird, I also updated my comfy instance and this does not shows correctly.

**Comment by Trung0246** [comment]
Hm weird, I also updated my comfy instance and this does shows correctly.

Maybe a conflict with other node pack.

**Comment by Trung0246** [comment]
Nvm I know why now. The current Hub did not take account into new menu UI.

**Comment by JorgeR81** [comment]
I disabled all my other custom nodes, but I still have this issue. 
I tried in Chrome and Edge.
I'm on Windows 10.

---

I also tried the alternative DOM implementation.

![t00](https://github.com/user-attachments/assets/b0a8b608-e500-4a05-94df-d4f198a68741)


It's a little better, but it also has issues.
The text boxes are still not shown in the nodes.
The text boxes are placed correctly in the Hub, but they are not usable.
When I click on the text box in the Hub, it disappears, and it shows up in the node instead !

![t2](https://github.com/user-attachments/assets/96e46d5b-0df9-4656-8d46-6d188082e2a6)

**Comment by Trung0246** [comment]
Yeah past version of comfyui did make text box editable in Hub. For some reason the current comfyui tried to refocus and shift the textbox widget to original node. Honestly this is really hard to fix so it's unlikely I'll touch it soon until I can find out what's keep shifting the textbox back to original location.

Will leave this issue open and see if I can figure out the root cause.

**Comment by JorgeR81** [comment]
With the current frontend, the text field it's placed correctly, in the Hub node, even without enabling the alternative DOM widget.

But we still have the other issue, where the text field shifts back to the original node when we click in the hub node.
